css/homepage.css

```diff
@@ -0,0 +1,75 @@
+.main-head.ecom-navbar {
+  background: var(--primary-color);
+  position: sticky;
+  top: 0px;
+  z-index: 3;
```

Comment:
Lateron there might be stacking context conflicts, best to keep navbar z-index at highest possible value like 9999